app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -81,7 +81,7 @@
     private boolean itemsLoaded = false;
     private boolean viewsCreated = false;
 
-    private AtomicReference<MainActivity> activity = new AtomicReference<MainActivity>();
+    private AtomicReference<MainActivity> activity = new AtomicReference<>();
```

Comment:
Actually, I don't really think we even want to hold an AtomicReference here. This is a context leak about to happen...